### ISSUE TITLE
CSVSequenceRecordReader adds 1 dimension to data vector

### ISSUE THREAD

**Issue by Arihant-Joshi** [deeplearning4j/deeplearning4j#8463]
#### Issue Description
Hi
I am trying to train an RNN for intent detection.
I have converted the words to vectors, and split them into separate files as required by dl4j.
This is a problem of MultiLabel Classification, so I am basically training a different classifier model for each intent.
But on model.fit(), I am getting the following error-
[StackTrace](https://gist.github.com/Arihant-Joshi/2278f5de42100a53dfc7d6e5eff60124)

Here is the function I am using for training-
[trainIntent.java](https://gist.github.com/Arihant-Joshi/7e8b323f1d32638cf8797daf01a900b2)

[Sample features file](https://gist.github.com/Arihant-Joshi/ff3d8ec4fb41199f9b51c57b8c6cd957)
[Sample Labels file](https://gist.github.com/Arihant-Joshi/4edf5de55c85b907dd176f9d675a3158)

It seems the issue is that my Model expects an output vector of dimensions [1,2], but it reads the output from my file as having dimension [1,2,1].
I have checked [DL4JExamples UCISequenceClassificationExample](https://github.com/eclipse/deeplearning4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/recurrent/seqclassification/UCISequenceClassificationExample.java), and the label files it uses seem similar to mine, so I dont know why CSVSequenceRecordReader() is reading my files differently.

#### Version Information

I have the following setup-
* dl4j version - 1.0.0-SNAPSHOT
* Windows 10
*Java 1.8.0_102
*Eclipse 2019-06 (4.12.0)
*[POM.xml](https://gist.github.com/Arihant-Joshi/30a721ff95ae9a6729bdea2305eeea6f)

**Comment by Arihant-Joshi** [comment]
This was solved by changing the output layer from OutputLayer.Builder to RnnOutputLayer.Builder.
Kindly delete this issue.
